python/pyarrow/includes/libarrow_fs.pxd

```diff
@@ -1,92 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Any particular reason for removing this file? I think it was fine to have a separate declarations file (also `libarrow.pxd` is really getting big).

Reply:
Again, I was trying to follow the C++ namespaces here. The non-optional C++ module headers are all placed in `libarrow.pxd`

Reply:
Yes. I wouldn't mind some separation though. It's a bit painful to navigate through `libarrow.pxd`.

Reply:
Create JIRA https://issues.apache.org/jira/browse/ARROW-6673